// validator/es/index.js
import toDate from 'validator/es/lib/toDate';
import toFloat from 'validator/es/lib/toFloat';
import toInt from 'validator/es/lib/toInt';
import toBoolean from 'validator/es/lib/toBoolean';
import equals from 'validator/es/lib/equals';
import contains from 'validator/es/lib/contains';
import matches from 'validator/es/lib/matches';
import isEmail from 'validator/es/lib/isEmail';
import isURL from 'validator/es/lib/isURL';
import isMACAddress from 'validator/es/lib/isMACAddress';
import isIP from 'validator/es/lib/isIP';
import isIPRange from 'validator/es/lib/isIPRange';
import isFQDN from 'validator/es/lib/isFQDN';
import isDate from 'validator/es/lib/isDate';
import isTime from 'validator/es/lib/isTime';
import isBoolean from 'validator/es/lib/isBoolean';
import isLocale from 'validator/es/lib/isLocale';
import isAbaRouting from 'validator/es/lib/isAbaRouting';
import isAlpha, { locales as isAlphaLocales } from 'validator/es/lib/isAlpha';
import isAlphanumeric, { locales as isAlphanumericLocales } from 'validator/es/lib/isAlphanumeric';
import isNumeric from 'validator/es/lib/isNumeric';
import isPassportNumber from 'validator/es/lib/isPassportNumber';
import isPort from 'validator/es/lib/isPort';
import isLowercase from 'validator/es/lib/isLowercase';
import isUppercase from 'validator/es/lib/isUppercase';
import isIMEI from 'validator/es/lib/isIMEI';
import isAscii from 'validator/es/lib/isAscii';
import isFullWidth from 'validator/es/lib/isFullWidth';
import isHalfWidth from 'validator/es/lib/isHalfWidth';
import isVariableWidth from 'validator/es/lib/isVariableWidth';
import isMultibyte from 'validator/es/lib/isMultibyte';
import isSemVer from 'validator/es/lib/isSemVer';
import isSurrogatePair from 'validator/es/lib/isSurrogatePair';
import isInt from 'validator/es/lib/isInt';
import isFloat, { locales as isFloatLocales } from 'validator/es/lib/isFloat';
import isDecimal from 'validator/es/lib/isDecimal';
import isHexadecimal from 'validator/es/lib/isHexadecimal';
import isOctal from 'validator/es/lib/isOctal';
import isDivisibleBy from 'validator/es/lib/isDivisibleBy';
import isHexColor from 'validator/es/lib/isHexColor';
import isRgbColor from 'validator/es/lib/isRgbColor';
import isHSL from 'validator/es/lib/isHSL';
import isISRC from 'validator/es/lib/isISRC';
import isIBAN, { locales as ibanLocales } from 'validator/es/lib/isIBAN';
import isBIC from 'validator/es/lib/isBIC';
import isMD5 from 'validator/es/lib/isMD5';
import isHash from 'validator/es/lib/isHash';
import isJWT from 'validator/es/lib/isJWT';
import isJSON from 'validator/es/lib/isJSON';
import isEmpty from 'validator/es/lib/isEmpty';
import isLength from 'validator/es/lib/isLength';
import isByteLength from 'validator/es/lib/isByteLength';
import isUUID from 'validator/es/lib/isUUID';
import isMongoId from 'validator/es/lib/isMongoId';
import isAfter from 'validator/es/lib/isAfter';
import isBefore from 'validator/es/lib/isBefore';
import isIn from 'validator/es/lib/isIn';
import isLuhnNumber from 'validator/es/lib/isLuhnNumber';
import isCreditCard from 'validator/es/lib/isCreditCard';
import isIdentityCard from 'validator/es/lib/isIdentityCard';
import isEAN from 'validator/es/lib/isEAN';
import isISIN from 'validator/es/lib/isISIN';
import isISBN from 'validator/es/lib/isISBN';
import isISSN from 'validator/es/lib/isISSN';
import isTaxID from 'validator/es/lib/isTaxID';
import isMobilePhone, { locales as isMobilePhoneLocales } from 'validator/es/lib/isMobilePhone';
import isEthereumAddress from 'validator/es/lib/isEthereumAddress';
import isCurrency from 'validator/es/lib/isCurrency';
import isBtcAddress from 'validator/es/lib/isBtcAddress';
import { isISO6346, isFreightContainerID } from 'validator/es/lib/isISO6346';
import isISO6391 from 'validator/es/lib/isISO6391';
import isISO8601 from 'validator/es/lib/isISO8601';
import isRFC3339 from 'validator/es/lib/isRFC3339';
import isISO31661Alpha2 from 'validator/es/lib/isISO31661Alpha2';
import isISO31661Alpha3 from 'validator/es/lib/isISO31661Alpha3';
import isISO4217 from 'validator/es/lib/isISO4217';
import isBase32 from 'validator/es/lib/isBase32';
import isBase58 from 'validator/es/lib/isBase58';
import isBase64 from 'validator/es/lib/isBase64';
import isDataURI from 'validator/es/lib/isDataURI';
import isMagnetURI from 'validator/es/lib/isMagnetURI';
import isMailtoURI from 'validator/es/lib/isMailtoURI';
import isMimeType from 'validator/es/lib/isMimeType';
import isLatLong from 'validator/es/lib/isLatLong';
import isPostalCode, { locales as isPostalCodeLocales } from 'validator/es/lib/isPostalCode';
import ltrim from 'validator/es/lib/ltrim';
import rtrim from 'validator/es/lib/rtrim';
import trim from 'validator/es/lib/trim';
import escape from 'validator/es/lib/escape';
import unescape from 'validator/es/lib/unescape';
import stripLow from 'validator/es/lib/stripLow';
import whitelist from 'validator/es/lib/whitelist';
import blacklist from 'validator/es/lib/blacklist';
import isWhitelisted from 'validator/es/lib/isWhitelisted';
import normalizeEmail from 'validator/es/lib/normalizeEmail';
import isSlug from 'validator/es/lib/isSlug';
import isLicensePlate from 'validator/es/lib/isLicensePlate';
import isStrongPassword from 'validator/es/lib/isStrongPassword';
import isVAT from 'validator/es/lib/isVAT';
var version = '13.12.0';
var validator = {
  version: version,
  toDate: toDate,
  toFloat: toFloat,
  toInt: toInt,
  toBoolean: toBoolean,
  equals: equals,
  contains: contains,
  matches: matches,
  isEmail: isEmail,
  isURL: isURL,
  isMACAddress: isMACAddress,
  isIP: isIP,
  isIPRange: isIPRange,
  isFQDN: isFQDN,
  isBoolean: isBoolean,
  isIBAN: isIBAN,
  isBIC: isBIC,
  isAbaRouting: isAbaRouting,
  isAlpha: isAlpha,
  isAlphaLocales: isAlphaLocales,
  isAlphanumeric: isAlphanumeric,
  isAlphanumericLocales: isAlphanumericLocales,
  isNumeric: isNumeric,
  isPassportNumber: isPassportNumber,
  isPort: isPort,
  isLowercase: isLowercase,
  isUppercase: isUppercase,
  isAscii: isAscii,
  isFullWidth: isFullWidth,
  isHalfWidth: isHalfWidth,
  isVariableWidth: isVariableWidth,
  isMultibyte: isMultibyte,
  isSemVer: isSemVer,
  isSurrogatePair: isSurrogatePair,
  isInt: isInt,
  isIMEI: isIMEI,
  isFloat: isFloat,
  isFloatLocales: isFloatLocales,
  isDecimal: isDecimal,
  isHexadecimal: isHexadecimal,
  isOctal: isOctal,
  isDivisibleBy: isDivisibleBy,
  isHexColor: isHexColor,
  isRgbColor: isRgbColor,
  isHSL: isHSL,
  isISRC: isISRC,
  isMD5: isMD5,
  isHash: isHash,
  isJWT: isJWT,
  isJSON: isJSON,
  isEmpty: isEmpty,
  isLength: isLength,
  isLocale: isLocale,
  isByteLength: isByteLength,
  isUUID: isUUID,
  isMongoId: isMongoId,
  isAfter: isAfter,
  isBefore: isBefore,
  isIn: isIn,
  isLuhnNumber: isLuhnNumber,
  isCreditCard: isCreditCard,
  isIdentityCard: isIdentityCard,
  isEAN: isEAN,
  isISIN: isISIN,
  isISBN: isISBN,
  isISSN: isISSN,
  isMobilePhone: isMobilePhone,
  isMobilePhoneLocales: isMobilePhoneLocales,
  isPostalCode: isPostalCode,
  isPostalCodeLocales: isPostalCodeLocales,
  isEthereumAddress: isEthereumAddress,
  isCurrency: isCurrency,
  isBtcAddress: isBtcAddress,
  isISO6346: isISO6346,
  isFreightContainerID: isFreightContainerID,
  isISO6391: isISO6391,
  isISO8601: isISO8601,
  isRFC3339: isRFC3339,
  isISO31661Alpha2: isISO31661Alpha2,
  isISO31661Alpha3: isISO31661Alpha3,
  isISO4217: isISO4217,
  isBase32: isBase32,
  isBase58: isBase58,
  isBase64: isBase64,
  isDataURI: isDataURI,
  isMagnetURI: isMagnetURI,
  isMailtoURI: isMailtoURI,
  isMimeType: isMimeType,
  isLatLong: isLatLong,
  ltrim: ltrim,
  rtrim: rtrim,
  trim: trim,
  escape: escape,
  unescape: unescape,
  stripLow: stripLow,
  whitelist: whitelist,
  blacklist: blacklist,
  isWhitelisted: isWhitelisted,
  normalizeEmail: normalizeEmail,
  toString: toString,
  isSlug: isSlug,
  isStrongPassword: isStrongPassword,
  isTaxID: isTaxID,
  isDate: isDate,
  isTime: isTime,
  isLicensePlate: isLicensePlate,
  isVAT: isVAT,
  ibanLocales: ibanLocales
};
export default validator;
